File: src/targets/php/guzzle/fixtures/multipart-form-data-no-params.php
<?php
<<<<<<< HEAD
require_once('vendor/autoload.php');

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'https://httpbin.org/anything', [
=======

$client = new \GuzzleHttp\Client();

$response = $client->request('POST', 'http://mockbin.com/har', [
>>>>>>> upstream/master
  'headers' => [
    'Content-Type' => 'multipart/form-data',
  ],
]);

echo $response->getBody();

File: src/targets/clojure/clj_http/fixtures/multipart-form-data-no-params.clj
(require '[clj-http.client :as client])

<<<<<<< HEAD
(client/post "https://httpbin.org/anything" {:headers {:Content-Type "multipart/form-data"}})
=======
(client/post "http://mockbin.com/har" {:headers {:Content-Type "multipart/form-data"}})
>>>>>>> upstream/master
